requirements_test.txt

```diff
@@ -1,7 +1,7 @@
 -r requirements_test_pre_commit.txt
 -r requirements_test_min.txt
 coverage~=7.1
-pre-commit~=2.21
+pre-commit~=3.0
```

Comment:
I'm wondering why pre-commit is in the test requirements and not the pre-commit requirements. The current fail for 3.7 is due to it, but I don't think we actually use pre-commit in pytest.